scenarios/scenario-04/README.md

```diff
@@ -0,0 +1,16 @@
+# Scenario: 
+
```

Comment:
Flesh out the scenario description and operating paradigms / constraints.

Reply:
will create a separate PR for this, where will rename it to persistence and move all app persistence layer to it